### PR TITLE
Add a header

### PR DESCRIPTION
**Pull request by stacymbush**

### CODE REVIEW

**Comment by jhalljhall** [comment]
Hey Stacy!  I can access your code and see what is wrong.

**Comment by stacymbush** [comment]
> Hey Stacy! I can access your code and see what is wrong.

Super!  Thank you!

File: index.html [hunk 1]
```diff
@@ -2,10 +2,9 @@
...
     <title>Sea of Awesomeness</title>
   </head>
-    <head>
-        <title>My awesome web page</title>
-    </head>
-
-    Hello there, awesome person!
+    <body>
+        <h1>My awesome web page</h1>
+    <img src+"https://avatars0.githubusercontent.com/u/19331862?v=4
```
Comment (jhalljhall) [inline index.html]: @stacymbush you should have this in your code for the image:

``

You have:

`<img src+"https://avatars0.githubusercontent.com/u/19331862?v=4`

Comment (stacymbush) [reply index.html]: Well shoot.

Comment (stacymbush) [reply index.html]: So do I just go back to the Code tab and update it?

Comment (stacymbush) [reply index.html]: And what does "Resolve conversation" do

Comment (jhalljhall) [reply index.html]: Yes.  You can go back to the code tab and edit the file.

Comment (jhalljhall) [reply index.html]: You can resolve the conversation too.  It will take our convo and archive it.

**Review by jhalljhall — APPROVED** [review]
This looks much better.

**Review by jhalljhall — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -2,10 +2,9 @@
...
     <title>Sea of Awesomeness</title>
   </head>
-    <head>
-        <title>My awesome web page</title>
-    </head>
-
-    Hello there, awesome person!
+    <body>
+        <h1>My awesome web page</h1>
+    <img src="https://avatars0.githubusercontent.com/u/19331862?v=4
```
Comment (jhalljhall) [inline index.html]: @stacymbush 

You are missing a "> at the end of the  tag.

You have:
``

Comment (jhalljhall) [reply index.html]: see the ending "> that is needed?

**Review by stacymbush — COMMENTED** [review]
Add a bracket to the end of the image link

File: index.html (hunk 2 above)
Comment (stacymbush) [reply index.html]: I see it and I added it but nothing seems to be happening.

**Pull request merged**